addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -27,6 +27,7 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
                 client: this._barcodeClientAction,
                 discount: this._barcodeDiscountAction,
                 error: this._barcodeErrorAction,
+                package: this._barcodeProductAction,
```

Comment:
this is here that you probably need to have an action that will add the quantity of the package of the scanned product to the PO